### DISCUSSION TITLE
Copilot AI pass

### DISCUSSION THREAD

**Discussion by piksl1** [community/community#79262] [GitHub Education]
### Hi everyone,

hi , i'm a self-learning (beginner) and i want to know if there is a way i can use Copilot AI as a student for studying purposes clearly

**Comment by ghost** [comment]
Hey @piksl1😀,

You can definitely 💯 use **GitHub  CoPilot**  for free if  you are a **Verified Student 🧑‍🎓or Teacher 🧑‍🏫** and you have a **GitHub Student Pack**.
You can simply start using CoPilot for your Project and study purposes.
You just need to integrate **CoPilot** with any IDE like **Visual Studio Code**  by logging in VS Code with GitHub account and Install GitHub Extension.
> Welcome 😁
> Keep Contributing Open-Source🧑‍💻
> @abhinavkumar2369

![image](https://github.com/community/community/assets/148275310/666b4014-c5f2-4c6c-85bd-2ecf24581d90)

**Comment by kurtgrung** [comment]
Try this free VS Code extension - "Google Cloud" has Duet AI built into it. 

[Google Cloud Code + Duet AI](https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.cloudcode)
